Dockerfile

```diff
@@ -1,35 +1,37 @@
-FROM ubuntu-debootstrap:14.04
+FROM php:7.0.7-alpine
 MAINTAINER Christian Lück <christian@lueck.tv>
 
-RUN DEBIAN_FRONTEND=noninteractive apt-get update && apt-get install -y \
-  nginx supervisor php5-fpm php5-cli \
-  php5-pgsql php5-mysql php5-sqlite php5-mssql \
-  wget
-
-# add adminer as the only nginx site
-ADD adminer.nginx.conf /etc/nginx/sites-available/adminer
-RUN ln -s /etc/nginx/sites-available/adminer /etc/nginx/sites-enabled/adminer
-RUN rm /etc/nginx/sites-enabled/default
-
-# install adminer and default theme
-RUN mkdir /var/www
-RUN wget http://www.adminer.org/latest.php -O /var/www/index.php
-RUN wget https://raw.github.com/vrana/adminer/master/designs/hever/adminer.css -O /var/www/adminer.css
-WORKDIR /var/www
-RUN chown www-data:www-data -R /var/www
+## Install php and dependencies
+RUN set -ex \
+    && apk add --no-cache postgresql-dev sqlite-dev \
+    && docker-php-ext-install pdo pdo_sqlite pdo_mysql pdo_pgsql
 
-# tune PHP settings for uploading large dumps
-RUN echo "upload_max_filesize = 2000M" >> /etc/php5/upload_large_dumps.ini \
- && echo "post_max_size = 2000M"       >> /etc/php5/upload_large_dumps.ini \
- && echo "memory_limit = -1"           >> /etc/php5/upload_large_dumps.ini \
- && echo "max_execution_time = 0"      >> /etc/php5/upload_large_dumps.ini \
- && ln -s ../../upload_large_dumps.ini /etc/php5/fpm/conf.d \
- && ln -s ../../upload_large_dumps.ini /etc/php5/cli/conf.d
+## php odbc hack (@see https://github.com/docker-library/php/issues/103)
+RUN set -x \
+    && cd /usr/src/php/ext/odbc \
+    && apk add --no-cache unixodbc-dev $PHPIZE_DEPS \
+    && phpize \
+    && sed -ri 's@^ *test +"\$PHP_.*" *= *"no" *&& *PHP_.*=yes *$@#&@g' configure \
+    && docker-php-ext-configure odbc --with-unixODBC=shared,/usr \
+    && docker-php-ext-install odbc \
+    && apk del $PHPIZE_DEPS
 
-# expose only nginx HTTP port
-EXPOSE 80
+## Add Tini
+RUN apk add --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/edge/community/ tini
+ENTRYPOINT ["/usr/bin/tini", "--"]
+
+## Add the files
+ADD php.ini /usr/local/etc/php/conf.d/php.ini
+
+## install adminer and default theme
+ENV ADMINER_VERSION 4.2.4
```

Comment:
@kusmierz See the discussion here: https://github.com/clue/docker-adminer/issues/5